{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"requests.exceptions.ConnectionError: HTTPConnectionPool(host='grafana-0.grafana-endpoints.cos-lite.svc.cluster.local', port=3000): Max retries exceeded with url: /api/search?query=5G%20Network%20Overview (Caused by NameResolutionError(\"<urllib3.connection.HTTPConnection object at 0x7f0c647bb1c0>: Failed to resolve 'grafana-0.grafana-endpoints.cos-lite.svc.cluster.local' ([Errno -3] Temporary failure in name resolution)\"))","children":[{"name":"test_given_cos_lite_integrated_with_sdcore_when_searching_for_5g_network_overview_dashboard_in_grafana_then_dashboard_exists","uid":"740626595b3862f6","parentUid":"d2e863ef7937e6cf13c7fdcb558c5a34","status":"broken","time":{"start":1741019743804,"stop":1741019745728,"duration":1924},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":true,"retriesCount":0,"retriesStatusChange":false,"parameters":[],"tags":["asyncio","abort_on_fail"]}],"uid":"d2e863ef7937e6cf13c7fdcb558c5a34"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}